packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -400,6 +404,36 @@ describe('Service dependency abstractions', () => {
       await provider.shutdown();
     });
 
+    it('should execute a provider error from the ogmios, and be intercepted by txSubmit', async () => {
+      provider = await getOgmiosTxSubmitProvider(dnsResolver, logger, {
+        ogmiosSrvServiceName: process.env.OGMIOS_SRV_SERVICE_NAME
+      });
+
+      await provider.initialize();
+      await provider.start();
+      expect(
+        provider.submitTx({
+          context: { handles: [getAliceHandleProviderResponse] },
+          signedTransaction: bufferToHexString(Buffer.from(new Uint8Array([])))
+        })
+      ).resolves.toEqual({ context: { handles: [getAliceHandleProviderResponse] } });
+    });
+
+    it('should execute a provider error from the ogmios provider and be intercepted by txSubmit', async () => {
```

Comment:
Try rewording to precisely describe the test, using my suggestion above as a guide.